### ISSUE TITLE
ValueError: cannot reshape array of size 1024 into shape (16,16)^

### ISSUE THREAD

**Issue by toastido** [overviewer/Minecraft-Overviewer#1688]
After updating and upgrading my worlds from 1.14.4 to 15, and upgrading overviewer to the latest from the debian repo, I now get this error immediately on startup:

```
$  /usr/bin/overviewer.py --config=/home/minecraft/overviewer.conf.py
2019-12-12 15:31:33  Welcome to Minecraft Overviewer!
2019-12-12 15:31:33  Generating textures...
2019-12-12 15:31:49 E An error has occurred. This may be a bug. Please let us know!
See http://docs.overviewer.org/en/latest/index.html#help

This is the error that occurred:
Traceback (most recent call last):
  File "/usr/bin/overviewer.py", line 650, in <module>
    ret = main()
  File "/usr/bin/overviewer.py", line 559, in main
    tileSetOpts.update({"spawn": w.find_true_spawn()})  # TODO find a better way to do this
  File "/usr/local/lib/python3.6/dist-packages/overviewer_core/world.py", line 219, in find_true_spawn
    chunk = regionset.get_chunk(chunkX, chunkZ)
  File "/usr/local/lib/python3.6/dist-packages/overviewer_core/world.py", line 1369, in get_chunk
    biomes = biomes.reshape((16,16))
ValueError: cannot reshape array of size 1024 into shape (16,16)
```
I have attached my config file to see if that helps.

System is:
Ubuntu 18.04.3 LTS
Python 3.6.9
Package is (from dpkg -l):
ii  minecraft-overviewer                  0.14.96-0~overviewer1                           amd64

[overviewer.conf.py.txt](https://github.com/overviewer/Minecraft-Overviewer/files/3956540/overviewer.conf.py.txt)

**Comment by CounterPillow** [comment]
Can you show me the output of `overviewer.py --version`? I suspect you have a conflicting local installation of overviewer that is out-of-date, as the line numbers do not match up with the current version.

**Comment by toastido** [comment]
Interesting.  
I just ran the upgrade via apt, and don't have an installation aside from the one from apt.  

Just to test, did an `apt remove minecraft-overviewer` and it removed the binary at /usr/bin/overviewer.py.
I then did an `apt install minecraft-overviewer`  and it installed the same version.

See below:

> root@bbminecraft:~# locate overviewer.py
> /usr/bin/overviewer.py
> root@bbminecraft:~# /usr/bin/overviewer.py --version
> 2019-12-12 16:06:11 W You are running Overviewer as root. It is recommended that you never do this, as it is dangerous for your system. If you are running into permission errors, fix your file/directory permissions instead. Overviewer does not need access to critical system resources and therefore does not require root access.
> Minecraft Overviewer 0.14.87 (7d9feb3)
> built on Tue Nov 19 03:56:46 2019
> root@bbminecraft:~# apt remove minecraft-overviewer
> Reading package lists... Done
> Building dependency tree
> Reading state information... Done
> The following packages will be REMOVED:
>   minecraft-overviewer
> 0 upgraded, 0 newly installed, 1 to remove and 0 not upgraded.
> After this operation, 1,516 kB disk space will be freed.
> Do you want to continue? [Y/n] y
> (Reading database ... 122961 files and directories currently installed.)
> Removing minecraft-overviewer (0.14.96-0~overviewer1) ...
> root@bbminecraft:~# ls -al /usr/bin/overviewer.py
> ls: cannot access '/usr/bin/overviewer.py': No such file or directory
> root@bbminecraft:~# apt install minecraft-overviewer
> Reading package lists... Done
> Building dependency tree
> Reading state information... Done
> The following NEW packages will be installed:
>   minecraft-overviewer
> 0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
> Need to get 371 kB of archives.
> After this operation, 1,516 kB of additional disk space will be used.
> Get:1 https://overviewer.org/debian ./ minecraft-overviewer 0.14.96-0~overviewer1 [371 kB]
> Fetched 371 kB in 1s (343 kB/s)
> Selecting previously unselected package minecraft-overviewer.
> (Reading database ... 122840 files and directories currently installed.)
> Preparing to unpack .../minecraft-overviewer_0.14.96-0~overviewer1_amd64.deb ...
> Unpacking minecraft-overviewer (0.14.96-0~overviewer1) ...
> Setting up minecraft-overviewer (0.14.96-0~overviewer1) ...
> root@bbminecraft:~# /usr/bin/overviewer.py --version
> 2019-12-12 16:07:37 W You are running Overviewer as root. It is recommended that you never do this, as it is dangerous for your system. If you are running into permission errors, fix your file/directory permissions instead. Overviewer does not need access to critical system resources and therefore does not require root access.
> Minecraft Overviewer 0.14.87 (7d9feb3)
> built on Tue Nov 19 03:56:46 2019


Looks like the deb in the repo is installing the wrong version.

**Comment by CounterPillow** [comment]
No, the .deb in the repo contains biome.py so it is the right version.

The key here is that the debian repos don't work with Python 3.6, so while /usr/bin/overviewer.py is from the package, it's not actually running the overviewer_core module from the package. It's running some locally installed version you have in `/usr/lib/python3.6/dist-packages`.

**Comment by toastido** [comment]
I don't have any such path `/user/lib/python3.6/dist-packages`

I do not have a locally installed version.

**Comment by CounterPillow** [comment]
Sorry, it's `/usr/local/lib/python3.6/dist-packages`. That's where your locally installed version is. That path is in your traceback too.

**Comment by toastido** [comment]
OK, so I removed those files and now it won't launch at all.  I took a look and it looks like it requires python3.5 which isn't installed via apt anymore for python3.  In that case, I uninstalled everything, downloaded the source, built and installed the source, and it appears to be working now.  

Thanks for the help.  (Maybe someone had installed from source prior to me taking over this system, which is why there were the 3.6 files.)
